rllib/core/rl_module/marl_module.py

```diff
@@ -11,6 +11,7 @@
 MultiAgentRLModuleConfig = MultiRLModuleConfig
 MultiAgentRLModuleSpec = MultiRLModuleSpec
 
+# ray.rllib.core.rl_module.rl_module.SingleAgentRLModuleSpec is also deprecated
```

Comment:
I did not know how to mark the alias as deprecated, so I just added a comment to the other related aliases.